e2e_tests/tests/api_utils.py

```diff
@@ -1,10 +1,17 @@
+from typing import Optional
+
 from determined.common import api
 from determined.common.api import authentication, certs
 from tests import config as conf
 
 
-def determined_test_session() -> api.Session:
+def determined_test_session(
```

Comment:
can we please somehow reconcile this helper and https://github.com/determined-ai/determined/blob/master/e2e_tests/tests/experiment/experiment.py#L322

Reply:
cause it's getting a bit chaotic: https://github.com/determined-ai/determined/pull/5548#discussion_r1080643582

Reply:
yeah makes sense. lets keep the one in api_utils. admin=true option is nice to have as well.

Reply:
I want to go ahead and land this w/o rerunning the tests and add this after. already working on it here: b8c818a8e24a18db88906d39867a8dc16bb43db3

Reply:
@hamidzr I made this change in my PR: https://github.com/determined-ai/determined/pull/5548.

Reply:
oh I have it up here too https://github.com/determined-ai/determined/pull/5772